xunit.runner.wpf/packages.config

```diff
@@ -1,18 +1,19 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="CommonServiceLocator" version="1.3" targetFramework="net452" />
-  <package id="MvvmLight" version="5.1.1.0" targetFramework="net452" />
-  <package id="MvvmLightLibs" version="5.1.1.0" targetFramework="net452" />
-  <package id="System.Collections" version="4.0.0" targetFramework="net46" />
+  <package id="MvvmLight" version="5.3.0.0" targetFramework="net46" />
```

Comment:
Who picked packages.config for this project.  Shouldn't we project.json all the things? :trollface: